src/Strategy.js

```diff
@@ -0,0 +1,73 @@
+//@flow
+
+import {expandStyle} from './compiler';
+import {Spec} from './ElementPropSpec';
```

Comment:
Looks like `resolve` should accept spec as its argument rather than consuming it globally. That way we can submit different specs for `<Element />` and `style()` components.